packages/flutter/test/material/stepper_test.dart

```diff
@@ -529,14 +529,15 @@ void main() {
     // which will change depending on where the test is run.
     expect(lines.length, greaterThan(7));
     expect(
-      lines.take(7).join('\n'),
+      lines.take(8).join('\n'),
       equalsIgnoringHashCodes(
         '══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞════════════════════════\n'
         'The following assertion was thrown building Stepper(dirty,\n'
         'dependencies: [_LocalizationsScope-[GlobalKey#00000]], state:\n'
         '_StepperState#00000):\n'
-        'Steppers must not be nested. The material specification advises\n'
-        'that one should avoid embedding steppers within steppers.\n'
+        'Steppers must not be nested.\n'
```

Comment:
fyi @goderbauer, @InMatrix   this test shows an intentional change caused by the CL.